solidity/ecdsa/contracts/api/IWalletRegistry.sol

```diff
@@ -38,4 +38,26 @@ interface IWalletRegistry {
 
     /// @notice Check current wallet creation state.
     function getWalletCreationState() external view returns (EcdsaDkg.State);
+
+    /// @notice Checks whether the given operator is a member of the given
+    ///         wallet signing group.
+    /// @param walletID ID of the wallet
+    /// @param walletMembersIDs Identifiers of the wallet signing group members
+    /// @param operator Address of the checked operator
+    /// @param walletMemberIndex Position of the operator in the wallet signing
+    ///        group members list
+    /// @return True - if the operator is a member of the given wallet signing
+    ///         group. False - otherwise.
+    /// @dev Requirements:
+    ///      - The `operator` parameter must be an actual sortition pool operator.
+    ///      - The expression `keccak256(abi.encode(walletMembersIDs))` must
+    ///        be exactly the same as the hash stored under `membersIdsHash`
+    ///        for the given `walletID`.
```

Comment:
We could mention the IDs are not stored in the contract for gas efficiency purposes but can be read from the chain by looking up `DkgResultSubmitted` and `DkgResultApproved` events.

Reply:
See https://github.com/keep-network/keep-core/pull/2925/commits/755214d5132100147b3185eab8833edfeeb0875d